pkgs/applications/audio/shuriken/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, qmake, alsaLib, jack2, rubberband, aubio, libsndfile
+, liblo, libsamplerate }:
+
+stdenv.mkDerivation rec {
```

Comment:
Please take a look at https://github.com/NixOS/nixpkgs/issues/65399. You likely want to include `mkDerivation` and use that directly, instead of from `stdenv`. In that case you can entirely drop the reference to `stdlib` and include `lib` instead.